glue/core/roi.py

```diff
@@ -55,6 +55,12 @@ def pixel_to_axes(axes, x, y):
     return axes.transAxes.inverted().transform(xy)
 
 
+def rotation(alpha):
+    """Return rotation matrix for angle alpha (increasing anticlockwise) around origin.
+    """
+    return np.array([[np.cos(alpha), -np.sin(alpha)], [np.sin(alpha), np.cos(alpha)]])
```

Comment:
This feels like something that belongs in ``glue.utils`` and should probably be called ``rotation_matrix`` or even ``rotation_matrix_2d`` in case we ever add 3-d rotations at some point.

Reply:
Just noted that there is already a `rotation_matrix` in `astropy.coordinates.matrix_utilities`, but it defaults to `u.deg` and the full 3-d case.